rand.go

```diff
@@ -4,6 +4,7 @@ import "math/rand"
 
 type Rand interface {
 	Intn(n int) int
+	Range(min, max int) int
```

Comment:
So far this interface has just been a subset of the methods of a `math/rand.Rand`. This new method does look pretty useful, but can you add a comment explaining what it does, boundary conditions, etc?

Reply:
done!

Reply:
@robbles do you feel that the comments adequately explain this now?